package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@nationalarchives/file-information",
-  "version": "1.0.311",
+  "version": "1.0.312",
```

Comment:
`package.json` is showing version as `1.0.313`?

Reply:
I've changed package.json to be 312. It doesn't need to be 313.